usaspending_api/api_contracts/contracts/award/idv/Funding.md

```diff
@@ -42,8 +42,8 @@ Returns File C financial data for an IDV (Indefinite Delivery Vehicle) award's d
 
 + Response 200 (application/json)
     + Attributes
-        + results (required, array[IDVFundingResponse])
-        + page_metadata (required, PageMetaDataObject, fixed-type)
+        + `results` (required, array[`IDVFundingResponse`])
```

Comment:
Is there a good reason to quote Objects?

Reply:
See above.  I have removed it as I have no strong opinion either way.